### PR TITLE
Feature/hw 2

### PR DESCRIPTION
**Pull request by LevapVeeskela**
finished HW-2

### CODE REVIEW

File: ItAcademyHomeWorks/ItAcademyHomeWorks.sln [hunk 1]
```diff
@@ -1,12 +1,52 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29306.81
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.852
 MinimumVisualStudioVersion = 10.0.40219.1
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/ItAcademyHomeWorks.sln]: why do not update your VS? :)

File: ItAcademyHomeWorks/Task1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            #region Explicit conversions
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Task1/Program.cs]: region - well done! I like it!

File: ItAcademyHomeWorks/Task1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            object o = i;   // boxing  - value on the heap
+            int j = (int)o; // unboxing - value on the stack
+
+            #endregion
+
+            Console.WriteLine($"pi: {doublePi}, newOnePi: {floatPi}, newTwoPi: {decimalPi}, newTreePi: {longPi}, longValue: {longValue}");
+            Console.ReadKey();
+        }
+    }
+}
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Task1/Program.cs]: well done

File: ItAcademyHomeWorks/Task2/Program.cs [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        private static void WriteInConsole(object[] data)
+        {
+            foreach (var value in data)
+            {
+                var nameType = value.GetType().FullName;
+                Console.Write($"{nameType} \n");
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Task2/Program.cs]: looks well!

File: ItAcademyHomeWorks/ItAcademyHomeWorks.sln (hunk 1 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/ItAcademyHomeWorks.sln]: I started to update VS on lesson, but I wasn't have good internet) this was last time

**Pull request merged**